### ISSUE TITLE
Allow GraphQL::Query::Arguments to be converted into a regular Ruby hash resolving aliases

### ISSUE THREAD

**Issue by gottfrois** [rmosolgo/graphql-ruby#684]
Based on https://github.com/rmosolgo/graphql-ruby/issues/65

Would be nice to have a method that returns a ruby hash with aliases resolved:

```ruby
args[:filter][:source_text][:$regex]
=> ".*logis.*"
args.to_h
=> {"filter"=>{"sourceText"=>{"regex"=>".*logis.*"}}}
```

As my argument is defined with an alias:

```ruby
argument :sourceText, ..., as: :source_text
```

This way I could plug the hash directly into a service object.

**Comment by rmosolgo** [comment]
That's a great suggestion, I think it should even be the default behavior of `to_h`

**Comment by rmosolgo** [comment]
This will require a bit of untangling in `Query::Arguments` and maybe event some API changes regarding how keys are returned to user code, so I'm going to target 1.6.0 for this!

**Comment by rmosolgo** [comment]
Fixed by #714
